### PR TITLE
GraphQL: Add categories query

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) adds the `categories` query.

## Affected DevDocs pages

https://devdocs.magento.com/guides/v2.4/graphql/queries/categories.html

whatsnew
Added the `categories` query to the _GraphQL Developer Guide_.

### CODE REVIEW

File: src/guides/v2.4/graphql/queries/categories.md [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        "total_pages": 2
+      }
+    }
+  }
+}
+```
+
+## Input attributes
+
+You must specify the `filters` attribute as input to your query.
```
Comment (danielrenaud) [inline src/guides/v2.4/graphql/queries/categories.md]: filters is not required. if it is omitted the store's default root category is returned (same as for categoryList)

File: src/guides/v2.4/graphql/queries/categories.md [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+## Syntax
+
+```graphql
+categories(filters: CategoryFilterInput pageSize: Int currentPage: Int): CategoryResult
+```
+
+## Example usage
+
+### Some
```
Comment (erikmarr) [inline src/guides/v2.4/graphql/queries/categories.md]:
```suggestion

```

**Comment by keharper** [comment]
running tests

**Comment by keharper** [comment]
running tests

**Pull request merged**